app/controllers/application_controller.rb

```diff
@@ -400,6 +400,16 @@ def filesystem_download
     send_data fs.contents, :filename => fs.name
   end
 
+  # Clear out search text when when requested by user
+  def clear_search_text
+    session[:flash_msgs] = @flash_array = nil           # clear out any messages from prior search results
+    # Clear out ALL iterations of prior search text
+    x_active_tree = @sb[:active_tree]
+    @search_text = @_params[:search_text] = @sb[:search_text] = @sb[:pol_search_text][x_active_tree] = nil
```

Comment:
@GregP you should be able to recreate this by clearing out all session data and try to clear search from one of the other controllers other than policy controller, i believe ``` @sb[:pol_search_text] ``` will not be set if user has not visited Policy Explorer hence the error.